doc/sphinx/language/cic.rst

```diff
@@ -1141,6 +1142,26 @@ Conversion is preserved as any (partial) instance :math:`I_j~q_1 … q_r` or
 :math:`C_i~q_1 … q_r` is mapped to the names chosen in the specific instance of
 :math:`\ind{p}{Γ_I}{Γ_C}`.
 
+.. warning::
+
+   The restriction that sorts are introduced by the inductive
+   declaration prevents inductive types declared in sections to
+   be template-polymorphic: they cannot parameterize over their
+   parameter's sorts, as these may be shared with other
+   definitions from the same section which can impose constraints on
+   them.
+
+.. flag:: Template Check
+
+   Unsetting option :flag:`Template Check` disables the check of
+   locality of the sorts when abstracting the inductive over its
+   parameters. This is an *unsafe* flag that can introduce
+   inconsistencies, it is only meant to help users incrementally update
+   code from Coq versions < 8.10 which did not implement this check. A
+   global ``-no-template-check`` command line option is also
+   available. Use at your own risk. Use of this flag is recorded in the
+   typing flags associated to a definition.
```

Comment:
And thus displayed by the `Print Assumption` command, right? If so, probably worth to say it explicitly.

Reply:
Actually it doesn't look like `Print Assumptions` print anything. Shouldn't it?

Reply:
Indeed it should, the accessors to the use of the template-check are there already.

Reply:
OK so now that `Print Assumptions` works, it should be mentioned in this description. Also, the flag is already deprecated, so it should appear in the description with the `.. deprecated:: 8.10` directive. BTW, what will coqchk say if this flag was used? Something similar to `Print Assumptions`, I suppose?